tests/ui/nll/user-annotations/region-error-ice-109072.stderr

```diff
@@ -21,6 +21,13 @@ help: consider introducing lifetime `'missing` here
 LL | impl<'missing> Lt<'missing> for () {
    |     ++++++++++
 
-error: aborting due to 2 previous errors
+error[E0277]: the trait bound `(): Lt<'_>` is not satisfied
```

Comment:
Ah, does this have to do with the fact that we do candidate filtering when they reference errors?

Reply:
> does this have to do with the fact that we do candidate filtering when they reference errors?
yes